src/types/list.rs

```diff
@@ -22,8 +22,7 @@ impl PyList {
     /// Construct a new list with the given elements.
     pub fn new<'p, T, U>(
         py: Python<'p>,
-        elements: impl IntoIterator<Item = T, IntoIter = U>,
-    ) -> &'p PyList
+        elements: impl IntoIterator<Item = T, IntoIter = U>) -> &'p PyList
```

Comment:
Does this need to be run through `cargo format`? It looks like the only change is whitespace, but I assume the original version was cargo formatted...

Reply:
I will run cargo format.